docusaurus/docs/extensions/api/nav/templates.md

```diff
@@ -0,0 +1,75 @@
+# Page templates
+
+When you create a page, either a [Custom Page](./custom-page.md) (`virtualType`) or a [Resource Page](./resource-page.md) (`configureType`), you can make use of different templates. By default it will use the `default` layout, but there are two different templates available:
```

Comment:
This tells users what is available but not why. It would be helpful to include a sentence in the introduction explaining why a user might want one template or the other.

Reply:
tbh, I think that once they get glimpse of the screenshots this is pretty self-explanatory. Either you need a sub-menu or not 😛